### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by sdanilv**
Pull request

### CODE REVIEW

File: jsonserver.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (VolodymyrKovalchuk88) [inline jsonserver.json]: what is this file for?

File: package.json [hunk 2]
```diff
@@ -3,21 +3,27 @@
...
   "private": true,
   "dependencies": {
+    "@ant-design/icons": "4.2.2",
     "@antv/g2plot": "1.1.21",
+    "@reduxjs/toolkit": "1.4.0",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "antd": "4.5.2",
     "axios": "0.19.2",
```
Comment (VolodymyrKovalchuk88) [inline package.json]: it occurred we are using fetch, can you migrate to simple fetch for us to pick up you plugin easier?

File: src/App.js [hunk 3]
```diff
@@ -1,151 +1,16 @@
...
-        <CartesianGrid strokeDasharray="3 3" />
-        <XAxis dataKey="date" />
-        <YAxis />
-        <Tooltip />
-        <Legend />
-        <Bar dataKey="count" fill="#8884d8" />
-      </BarChart>
-    </div>
+    <Provider store={store}>
+      <div className="App">
```
Comment (VolodymyrKovalchuk88) [inline src/App.js]: do we need this div and app class?

File: src/component/DatePanel.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  switch (dateType) {
+    case RANGE:
+      return (
+        <RangePicker
+          locale={locale}
+          onChange={onChangeRange}
+          format={"DD.MM.YYYY"}
+        />
+      );
+    case DAY:
```
Comment (VolodymyrKovalchuk88) [inline src/component/DatePanel.js:28]: and what about weeks?

File: src/component/Diagram.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  ]);
+  useEffect(() => {
+    if (endpoints.length) {
+      if (chartCount.current) chartCount.current.destroy();
+      chartCount.current = renderChart(endpoints, container.current, x, y);
+    }
+  }, [endpoints, x, y]);
+
+  return (
+    <div ref={container} style={{ padding:20, margin: 10,width:"100%" , height:"300px", border: "4px double black" }}>
```
Comment (VolodymyrKovalchuk88) [inline src/component/Diagram.js]: can we avoid inline styles? moving this to *.css?

File: src/redux/reducer.js [hunk 6]
```diff
@@ -0,0 +1,107 @@
+import {createSlice} from "@reduxjs/toolkit";
+import {getDataInRange, getEndpoints, getServices, getServicesName,} from "../api";
+import {MONTH, WEEK} from "../tools/constant";
+import {formattedDate} from "../tools/FomatedDate";
+
+const chartSlice = createSlice({
```
Comment (VolodymyrKovalchuk88) [inline src/redux/reducer.js]: let's get rid from redux, looks too complex for simple grid (use https://reactjs.org/docs/lifting-state-up.html if required), right?

File: jsonserver.json (hunk 1 above)
Comment (sdanilv) [reply jsonserver.json:1]: It file created with json-server . I use it for mock server

File: package.json (hunk 2 above)
Comment (sdanilv) [reply package.json]: migrated

File: jsonserver.json (hunk 1 above)
Comment (sdanilv) [reply jsonserver.json]: deleted

**Pull request merged**